lib/core/lib/generators/ros/resource/resource_generator.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../generators_helper.rb'
+
+module Ros
+  class ResourceGenerator < Rails::Generators::NamedBase
+    include GeneratorsHelper
+
+    def create_files
+      create_file "app/resources/#{name}_resource.rb", <<~FILE
```

Comment:
Created a resource file manually here as calling `invoke(:resource)` generates more files than just the resource file.

I can't find the file with a parent class of `Thor` which `resource`  is being defined to overwrite this behaviour.